app/models/order.rb

```diff
@@ -50,4 +81,12 @@ class Order < ActiveRecord::Base
   def formatted_order_date
     order_date.strftime("%-m/%-d/%Y") if order_date.present?
   end
+
+  def being_processed? # rubocop:disable Metrics/CyclomaticComplexity
+    pending? || approved? || rejected? || filled? || shipped? || received? || closed?
```

Comment:
Shipped, received, and closed don't seem to fit in here as "being processed".  Orders in these states are out of their hands at this point.